industry_restaurant/data/appointment_type.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="appointment_type_1" model="appointment.type">
+        <field name="name">Floor 1</field>
+        <field name="is_published" eval="True"/>
+        <field name="category">recurring</field>
+        <field name="cover_properties">{"background-image":"url(industry_restaurant/static/src/binary/appointment_resource/table1.jpg)","background_color_class":"o_cc3 o_cc","background_color_style":"","opacity":"0","resize_class":"o_half_screen_height o_record_has_cover","text_align_class":""}</field>
+        <field name="appointment_tz" model="res.users" eval="obj().env.ref('base.user_admin').tz or 'Europe/Brussels'"/>
+        <field name="schedule_based_on">resources</field>
+        <field name="min_cancellation_hours">0.25</field>
+        <field name="resource_total_capacity">4</field>
+        <field name="resource_manage_capacity" eval="True"/>
+        <field name="avatars_display">show</field>
+        <field name="website_published" eval="True"/>
+        <field name="slot_ids" eval="[
+            (0, 0, {'weekday': '1', 'start_hour': 9.0, 'end_hour': 12.0}),
+            (0, 0, {'weekday': '1', 'start_hour': 14.0, 'end_hour': 17.0}),
+            (0, 0, {'weekday': '2', 'start_hour': 9.0, 'end_hour': 12.0}),
+            (0, 0, {'weekday': '2', 'start_hour': 14.0, 'end_hour': 17.0}),
+            (0, 0, {'weekday': '3', 'start_hour': 9.0, 'end_hour': 12.0}),
+            (0, 0, {'weekday': '3', 'start_hour': 14.0, 'end_hour': 17.0}),
+            (0, 0, {'weekday': '4', 'start_hour': 9.0, 'end_hour': 12.0}),
+            (0, 0, {'weekday': '4', 'start_hour': 14.0, 'end_hour': 17.0}),
+            (0, 0, {'weekday': '5', 'start_hour': 9.0, 'end_hour': 12.0}),
+            (0, 0, {'weekday': '5', 'start_hour': 14.0, 'end_hour': 17.0}),
```

Comment:
I'd say it should be lunch and dinner time, no? Or is it timezone dependent?

Reply:
timezone dependent

Reply:
Ok but then it should be something like 12-14 and 19-21, no?